src/frontend/screens/ObservationEdit/ThumbnailAndActionTab.tsx

```diff
@@ -46,6 +49,11 @@ export const ThumbnailAndActionTab: FC<ThumbnailAndActionTab> = ({
   const {photos, audioRecordings, observationId} = usePersistedDraftObservation(
     store => store,
   );
+  const {openSheet, sheetRef, isOpen, closeSheet} = useBottomSheetModal({
```

Comment:
I would prefer if these variables referenced the audio permission sheet. Could we rename them? Here's a diff which you should be able to `git apply`:


```diff
diff --git a/src/frontend/screens/ObservationEdit/ThumbnailAndActionTab.tsx b/src/frontend/screens/ObservationEdit/ThumbnailAndActionTab.tsx
index bd9613ad..43ce793e 100644
--- a/src/frontend/screens/ObservationEdit/ThumbnailAndActionTab.tsx
+++ b/src/frontend/screens/ObservationEdit/ThumbnailAndActionTab.tsx
@@ -49,7 +49,12 @@ export const ThumbnailAndActionTab: FC<ThumbnailAndActionTab> = ({
   const {photos, audioRecordings, observationId} = usePersistedDraftObservation(
     store => store,
   );
-  const {openSheet, sheetRef, isOpen, closeSheet} = useBottomSheetModal({
+  const {
+    openSheet: openAudioPermissionSheet,
+    sheetRef: audioPermissionSheetRef,
+    isOpen: isAudioPermissionSheetOpen,
+    closeSheet: closeAudioPermissionSheet,
+  } = useBottomSheetModal({
     openOnMount: false,
   });
   const [permissionResponse] = Audio.usePermissions({request: false});
@@ -66,9 +71,9 @@ export const ThumbnailAndActionTab: FC<ThumbnailAndActionTab> = ({
     if (permissionResponse?.granted) {
       return navigation.navigate('Home', {screen: 'Map'});
     } else {
-      openSheet();
+      openAudioPermissionSheet();
     }
-  }, [navigation, openSheet, permissionResponse?.granted]);
+  }, [navigation, openAudioPermissionSheet, permissionResponse?.granted]);
 
   const bottomSheetItems = [
     {
@@ -106,9 +111,9 @@ export const ThumbnailAndActionTab: FC<ThumbnailAndActionTab> = ({
         <ActionTab items={bottomSheetItems} />
       </View>
       <PermissionAudio
-        closeSheet={closeSheet}
-        isOpen={isOpen}
-        sheetRef={sheetRef}
+        closeSheet={closeAudioPermissionSheet}
+        isOpen={isAudioPermissionSheetOpen}
+        sheetRef={audioPermissionSheetRef}
       />
     </>
   );
diff --git a/src/frontend/sharedComponents/PermissionAudio.tsx b/src/frontend/sharedComponents/PermissionAudio.tsx
index 4b43783f..e5b3fa27 100644
--- a/src/frontend/sharedComponents/PermissionAudio.tsx
+++ b/src/frontend/sharedComponents/PermissionAudio.tsx
@@ -8,7 +8,7 @@ import {BottomSheetModal} from './BottomSheetModal';
 import {Button} from './Button';
 import {Audio} from 'expo-av';
 import {useNavigationFromRoot} from '../hooks/useNavigationWithTypes';
-import {PermissionResponse} from 'expo-av/build/Audio';
+import {PermissionResponse, PermissionStatus} from 'expo-av/build/Audio';
 
 const handleOpenSettings = () => {
   Linking.openSettings();

```

Reply:
Okay, make sens, thanks